lib/charms/mongodb_libs/v0/mongodb.py

```diff
@@ -281,6 +281,14 @@ def update_user(self, config: MongoDBConfiguration):
             roles=self._get_roles(config),
         )
 
+    def set_user_password(self, username, password: str):
```

Comment:
```suggestion
    def rotate_user_password(self, username):
```

In line with comment above, can this be adjusted to just generate a password and return it?

Reply:
I hope it is resolved now https://github.com/delgod/mongodb-k8s-operator/pull/22#discussion_r933102683